.husky/pre-commit

```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run pre-commit-translate
```

Comment:
By removing these lines we are disabling key translation key extraction and linter on pre-commit, I think we need this.

Reply:
my bad this was a test